src/components/search-location/component.jsx

```diff
@@ -98,84 +98,89 @@ function SearchLocation({
+          [styles.mobile]: theme === 'mobile',
+          [styles.disabled]: disabled,
+          [className.inputContainer]: className.inputContainer,
         })}
-      />
-
-      {isSearchResultVisible &&
-        createPortal(
-          <div
-            ref={setPopperElement}
-            style={{ ...popperStyles.popper, width: parentWidth }}
-            className={styles.searchResultsList}
-            {...attributes.popper}
-          >
-            <ul
-              className={cx(styles.optionsList, {
-                [styles.fullWidth]: width === 'full',
-                [styles.mobile]: mobile,
-              })}
-              ref={searchOptionsListRef}
+        {...onboardingOverlay}
+      >
+        <input
+          type="text"
+          placeholder={placeholder || t('search')}
+          className={styles.input}
+          ref={inputRef}
+          onClick={handleOpenSearch}
+          onChange={handleInputChange}
+        />
+
+        <IconSearch
+          className={cx(styles.placeholderIcon, {
+            [className.placeholderIcon]: className.placeholderIcon,
+          })}
+        />
+
+        {isSearchResultVisible &&
+          createPortal(
+            <div
+              ref={setPopperElement}
+              style={{ ...popperStyles.popper, width: parentWidth }}
+              className={styles.searchResultsList}
+              {...attributes.popper}
             >
-              {searchResults.length > 0 ? (
-                searchResults.map((option) => (
-                  <li
-                    className={styles.option}
-                    key={option.key}
-                    onClick={() => {
-                      onOptionSelection(option);
-                      if (setSearcherOpen) {
-                        setSearcherOpen(false);
-                      }
-                      onNextonboardingStep(option);
-                    }}
-                  >
-                    {renderSuggestion(option.text)}
+              <ul
+                className={cx(styles.optionsList, {
+                  [styles.fullWidth]: width === 'full',
+                  [styles.mobile]: mobile,
+                })}
+                ref={searchOptionsListRef}
+              >
+                {searchResults.length > 0 ? (
+                  searchResults.map((option) => (
+                    <li
+                      className={styles.option}
+                      key={option.key}
+                      onClick={() => {
+                        onOptionSelection(option);
+                        if (setSearcherOpen) {
+                          setSearcherOpen(false);
+                        }
+                        onNextonboardingStep(option);
+                      }}
+                    >
+                      {renderSuggestion(option.text)}
+                    </li>
+                  ))
+                ) : (
+                  <li className={styles.emptyOption}>
```

Comment:
I think is because of these changes but not sure where. On desktop on the search on the right. When we search for something or we clear the input we get a white box that stays there: 

<img width="408" alt="image" src="https://user-images.githubusercontent.com/9701591/214003733-1408f874-b8f8-4e69-b67a-71f1dad6606d.png">